scripts.js

```diff
@@ -4,9 +4,7 @@ function getTriangleArea(a, h) {
 	} else {
```

Comment:
`else` is unnecessary, return in if will stop the code

your last return can be outside the if statement